### DISCUSSION TITLE
A Lite Version of the repo: Contains the minimum code and simple structure for people to understand [Ideally only the tweet integration & public node config]

### DISCUSSION THREAD

**Discussion by RedHorse823** [elizaOS/eliza#981] [Ideas]
Current eliza repo is way to complex that:
1. Have super complex file structure
2. Have way too many integration files and packages
3. Have way too many configs to fill

Ideally, we should have a lite repo that only:
1. Support Tweet Integration - Automated Post
2. Support a public node configs for AI model [Devs from certain regions could have a hard time getting openai/claude api keys]

**Comment by notwedtm** [comment]
hope this helps: https://github.com/ai16z/eliza-starter
